hr_operating_unit/models/hr_employee.py

```diff
@@ -0,0 +1,22 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrEmployee(models.Model):
+    _inherit = "hr.employee"
+
+    operating_unit_ids = fields.Many2many(
+        "operating.unit",
+        "operating_unit_employees_rel",
+        "employee_id",
+        "poid",
```

Comment:
This line was mentioned in the v12 PR that it shouldn't be needed in future versions.